rebar.config

```diff
@@ -1,11 +1,11 @@
 {erl_opts, [debug_info]}.
 {deps, [
-	{ergw, ".*", {git, "git://github.com/travelping/capwap", {tag, "1.8.0"}}},
+	{capwap, ".*", {git, "git://github.com/travelping/capwap", "master"}},
```

Comment:
this should be a tag off the capwap project.

Reply:
yes, but tag 1.9 does not have my changes..

Reply:
then you need to prepare a 1.9.1 release of capwap with you changes.